inlong-manager/manager-web/src/test/java/org/apache/inlong/manager/web/controller/openapi/SortControllerTest.java

```diff
@@ -131,4 +129,10 @@ private SortTaskSinkParamEntity prepareSinkParamsEntity(String task, String sink
                 .taskName(task)
                 .build();
     }
+
+    @Test
+    public void defaultTest() {
+        System.out.println("Online exception druid connection timeout cannot create transaction, "
```

Comment:
Please do not print the info to console, thanks.

Reply:
resolved